templates/base/src/hooks/Counter/Counter.js

```diff
@@ -0,0 +1,10 @@
+import { useState, useCallback } from 'react'
+function useCounter() {
```

Comment:
I am not sure how this hook will help in Unit testing??? The story was to add Hooks that help you in writing Unit Tests. For example. 

renderWithContext